server/src/middleware/errMiddleware.js

```diff
@@ -0,0 +1,14 @@
+export function notFound(req, res, next) {
+  res.status(404);
+  const error = new Error(`Not Found - ${req.originalUrl}`);
+  next(error);
+}
+
+export function errorHandler(err, req, res, next) {
+  const statusCode = res.statusCode !== 200 ? res.statusCode : 500;
```

Comment:
Could we not say `res.statusCode ? res.statusCode : 500`?

Reply:
@tsimian Thank you for your suggestion. I have made change accordingly.